c/flake.nix

```
@@ -0,0 +1,44 @@
+{
+  description = "Aux template for C project";
+
+  inputs.nixpkgs.url = "github:auxolotl/nixpkgs/nixos-unstable";
```

```suggestion
  inputs.nixpkgs.url = "github:auxolotl/nixpkgs";
```
we aren't actually updating branches besides master currently

---

master template use it too.

---

this should changed as well

---

This might need it's own dedicated issue